recipes/wxpython/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "wxPython" %}
+{% set version = "3.0.2.0" %}
+{% set sha256 = "d54129e5fbea4fb8091c87b2980760b72c22a386cb3b9dd2eebc928ef5e8df61" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   fn: wxPython-src-{{ version }}.tar.bz2  # [not win]
+   url: http://sourceforge.net/projects/wxpython/files/wxPython/{{ version }}/wxPython-src-{{ version }}.tar.bz2  # [not win]
+   sha256: {{ sha256 }}  # [not win]
+   fn: {{ version }}.zip  # [win]
+   url: https://github.com/NOAA-ORR-ERD/wxpython_wheels/archive/{{ version }}.zip  # [win]
```

Comment:
I believe this file is originally from Gohlke's website, correct?

Which one are you re-distributing?

```
wxPython‑3.0.2.0‑cp27‑none‑win32.whl
wxPython‑3.0.2.0‑cp27‑none‑win_amd64.whl
wxPython_common‑3.0.2.0‑py2‑none‑any.whl
``` 

Does Gohlke releases them under any license?

Can you please add a 1-line comment on this right above the URL.